wasm/src/lib.rs

```diff
@@ -209,7 +236,7 @@ where
                 // TODO: handle multiple errors
                 // https://github.com/microsoft/qsharp/issues/149
```

Comment:
I know this isn't part of your changes, but I'm pretty sure this todo can be removed. Evaluation can only ever return one error that caused evaluation to end, so the vector is known to be of size one. I missed this in my last refactor.